src/components/styles/homePage/homePage.module.css

```diff
@@ -0,0 +1,68 @@
+.carouselOuter {
+  margin-top: 24px;
+  width: 480px;
+  display: flex;
+  justify-content: center;
+}
+
+.bannerOuter {
+  margin-top: 24px;
+  margin-left: 5%;
+  width: 90%;
+  display: flex;
+}
+
+.carouselBadge {
+  width: 100%;
+  box-shadow: 8px 8px 8px 0px black;
+  border-radius: 20px 20px 0px 0px;
+  padding-top: 15px;
+  color: aliceblue;
+  text-align: center;
+  opacity: 0.8;
+  background-color: #404040;
+}
+
+.carouselItem {
+  background-color: #252525;
+}
+
+.badgeHeader {
+  color: rgb(241, 241, 241);
+  margin-bottom: 0;
+  padding-bottom: 1rem;
```

Comment:
Nie używaj `rem` i `em` do określania czekogolwiek innego niż wielkości czcionek, jest to jednostka relatywna do pierwszej stałej jednostki napisów w drzewie, oznacza to tyle że gdybyś chciał zmienić defaultową wielkość czcionki na całym screenie albo w całej w apce to to spowoduje że ci się zaczną paddingi, marginy gdziekolwiek indziej tego użyłeś rozjeżdżać.